requirements.in

```diff
@@ -3,7 +3,7 @@
 #    python -m scripts.install_third_party_libs
 #
 # to update `requirements.txt` and `third_party/python_libs` accordingly.
-apache-beam[gcp]==2.43.0
+apache-beam==2.54.0
```

Comment:
@chris7716 In this PR could you please also change 

google-cloud-ndb==2.3.4

Let's see if that works. Thanks.